src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InputFileResolverTests.cs

```diff
@@ -1,23 +1,27 @@
 ﻿using Shouldly;
 using Stryker.Core.Initialisation;
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.IO.Abstractions.TestingHelpers;
 using System.Reflection;
+using System.Runtime.InteropServices;
 using Xunit;
 
 namespace Stryker.Core.UnitTest.Initialisation
 {
     public class InputFileResolverTests
     {
+        private static readonly bool RunningOnWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         private string _currentDirectory { get; set; }
 
         public InputFileResolverTests()
         {
             _currentDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            Skip.IfNot(RunningOnWindows);
         }
 
-        [Fact]
+        [SkippableFact]
```

Comment:
What does this mean? Are these tests only ran on windows? I don't like that. 

Is there a clear path to make them run on linux in the near future?